### PR TITLE
Google storage driver

### PR DESCRIPTION
**Pull request by brenopessoa**
Initial commit for google storage driver.

Next steps :
* Implementations of  collections method's  on EntityManagerFactory or throw exception
* Implementation of Storage config to run locally or on google cloud depends configuration 
* Write tests.

I hope can help.
Cheers

### CODE REVIEW

File: google-storage-driver/.checkstyle [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (otaviojava) [inline google-storage-driver/.checkstyle]: could you remove this file?

File: google-storage-driver/.project [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (otaviojava) [inline google-storage-driver/.project]: ould you remove this file?

**Comment by otaviojava** [comment]
Thanks for your contribuition.
I belive the Google Storage driver also has support to document, isn't?

**Comment by brenopessoa** [comment]
I'm finished all the suggestions.

I think is possible and i'll investigate this, If you have any idea , please let me know.
Thank for your time and support.

**Comment by otaviojava** [comment]
Thanks

**Comment by otaviojava** [comment]
Thanks, I created this branch to we work on this driver: https://github.com/JNOSQL/diana-driver/tree/cloud_storage

**Comment by brenopessoa** [comment]
Ok , thank's.

**Pull request merged**